### PR TITLE
small doc fixes

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by mdellweg** [comment]
Fine with that, but it needs a rabase, and i know github does not notify you about that either. :(

**Comment by evgeni** [comment]
rebased

File: modules/katello_lifecycle_environment.py [hunk 1]
```diff
@@ -75,7 +75,7 @@
   katello_lifecycle_environment:
     username: "admin"
     password: "changeme"
-    server_url: "https://katello.example.com"
+    server_url: "https://foreman.example.com"
```
Comment (akofink) [inline modules/katello_lifecycle_environment.py]: I started using `katello.example.com` to specifically indicate that the server must have the Katello plugin, not just Foreman. I'd vote to scratch these `server_url` changes and perhaps even go ahead and change the `server_url` on the other `katello_` modules to `katello.example.com`.

Comment (evgeni) [inline modules/katello_lifecycle_environment.py]: Don't the names imply that Katello is required? But I'm fine with changing all katello modules to use katello.example.com too.

Comment (akofink) [inline modules/katello_lifecycle_environment.py]: Yeah, I guess that's true. Given that, I'm fine with keeping the `foreman.example.com` urls.

**Pull request merged**